src/core/util/env.js

```diff
@@ -15,7 +15,7 @@ export const devtools = inBrowser && window.__VUE_DEVTOOLS_GLOBAL_HOOK__
 // UA sniffing for working around browser-specific quirks
 const UA = inBrowser && window.navigator.userAgent.toLowerCase()
 const isIos = UA && /(iphone|ipad|ipod|ios)/i.test(UA)
-const iosVersionMatch = UA && isIos && UA.match(/os ([\d_]+)/)
+const iosVersionMatch = UA && isIos && /os ([\d_]+)/.test(UA)
```

Comment:
Ahh sorry! Totally misread it's context! All the tests ran fine though? Anyhow! I try! Thanks again for this amazing library!